packages/element-lib/src/sidetree-v2/___tests___/batching-spam-attack.js

```diff
@@ -0,0 +1,86 @@
+const { getCreatePayloadForKeyIndex } = require('../test-utils');
+const { MnemonicKeySystem } = require('../../../index');
+const element = require('../../../index');
+
+class Attacker {
+  constructor(sidetree, numberOfOperations) {
+    this.mks = new MnemonicKeySystem(MnemonicKeySystem.generateMnemonic());
+    this.sidetree = sidetree;
+    this.numberOfOperations = numberOfOperations;
+  }
+
+  async start() {
+    const operationNumbers = [...Array(this.numberOfOperations).keys()];
+    const promises = operationNumbers.map(async (index) => {
+      const createPayload = await getCreatePayloadForKeyIndex(this.mks, index);
+      const didUniqueSuffix = this.sidetree.func.getDidUniqueSuffix(createPayload);
+      return { didUniqueSuffix, operationBuffer: createPayload };
+    });
+    this.queue = await Promise.all(promises);
+    await this.sidetree.db.write('operationQueue', { queue: this.queue });
+  }
+}
+
+const db = new element.adapters.database.ElementRXDBAdapter({
+  name: 'element-test',
+});
+
+const storage = element.storage.ipfs.configure({
+  multiaddr: '/ip4/127.0.0.1/tcp/5001',
+});
+
+const blockchain = element.blockchain.ethereum.configure({
+  mnemonic: 'hazard pride garment scout search divide solution argue wait avoid title cave',
```

Comment:
nit: could pull from config

Reply:
I thought it was more convenient that the file was not linked to any config, that way we can just run it using `node batching-spam-attack.js` to try on a different machine

Reply:
this is a sound argument.